<?xml version="1.0"?>
<launch>
    <group ns="align_and_shoot">
        <node name="align_and_shoot_2024" pkg="behaviors" type="2024_align_and_shoot.py" output="screen"/>
    </group>
</launch>